src/CrowdedMod/Patches/GenericPatches.cs

```diff
@@ -159,15 +150,18 @@ public static void Prefix(ref GameSettingMenu __instance)
     //     }
     // }
 
-    [HarmonyPatch(typeof(GameOptionsMenu), nameof(GameOptionsMenu.Start))]
-    public static class GameOptionsMenu_Start
+    [HarmonyPatch(typeof(GameOptionsMenu), nameof(GameOptionsMenu.Initialize))]
+    public static class GameOptionsMenu_Initialize
     {
-        public static void Postfix(ref GameOptionsMenu __instance)
+        public static void Postfix(GameOptionsMenu __instance)
         {
-            __instance.GetComponentsInChildren<NumberOption>()
-                .First(o => o.Title == StringNames.GameNumImpostors)
-                // ReSharper disable once PossibleLossOfFraction
-                .ValidRange = new FloatRange(1, CrowdedModPlugin.MaxImpostors);
+            var numberOptions = __instance.GetComponentsInChildren<NumberOption>();
+
+            if (numberOptions.Any(o => o.Title == StringNames.GameNumImpostors))
+            {
+                numberOptions.First(o => o.Title == StringNames.GameNumImpostors)
```

Comment:
this iterates children twice which is an unnecessary performance loss, especially in heavily modded instances. You're free to worry about it not being present (we should've checked for that before lol), but use `FirstOrDefault` and check for null to iterate only once